firmware/stackchan/drivers/rs30x.ts

```
@@ -0,0 +1,318 @@
+                // trace(`got response for ${id}. triggering callback \n`)
+                this.#callbacks.get(id)?.(Array.from(rxBuf.slice(7, this.#idx - 1)))
+              } else {
+                trace(`unknown packet for ${id} ... ${rxBuf.slice(0, this.#idx)}. ignoring\n`)
+              }
+              this.#idx = 0
+              this.#state = RX_STATE.SEEK
+            }
+            break
+          default:
+            // @ts-ignore 6113
+            let _state: never
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)